spark/v3.2/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRemoveOrphanFilesProcedure.java

```diff
@@ -360,4 +360,29 @@ public void testRemoveOrphanFilesWithDeleteFiles() throws Exception {
         .collectAsList();
     Assert.assertEquals("Rows must match", records, actualRecords);
   }
+
+  @Test
+  public void testRemoveOrphanFilesWithStreamResultsEnabled() throws Exception {
+    sql("CREATE TABLE %s (id bigint NOT NULL) USING iceberg", tableName);
+    sql("INSERT INTO TABLE %s VALUES (1)", tableName);
+    Dataset<Row> df = spark.sql(String.format("select * from %s", tableName));
+    Table table = Spark3Util.loadIcebergTable(spark, tableName);
+    df.write().parquet(table.location() + "/data/orphan_files");
+    Thread.sleep(1000);
+    Timestamp currentTimestamp = Timestamp.from(Instant.ofEpochMilli(System.currentTimeMillis()));
+    List<Object[]> output1 = sql(
```

Comment:
Follow meaningful variable names，change `output1 `variable `dryrunOutput `instead, change `output2 `variable `streamResultsOutput `instead

Reply:
@felixYyu change `output1` -> `dryrunOutput `, `output2` -> `streamResultsOutput ` , also change `result1` -> `dryRunResult`, `result2` -> `streamResultsResult` in 'org.apache.iceberg.spark.actions.TestRemoveOrphanFilesAction#testRemoveOrphanFilesWithStreamResultsEnabled'